libcxx/include/fstream

```diff
@@ -778,7 +778,12 @@ basic_filebuf<_CharT, _Traits>* basic_filebuf<_CharT, _Traits>::close() {
     if (fclose(__h.release()))
       __rt = nullptr;
     __file_ = nullptr;
-    setbuf(0, 0);
+    // Reset the get and the put areas without nonetheless getting rid of the
+    // underlying buffers, which might have been configured by the user who might
+    // reopen the stream.
```

Comment:
```suggestion
    // Reset the get and the put areas without getting rid of the
    // underlying buffers, which might have been configured by the user.
    // Make sure to keep the buffer, since the user may re-open the stream.
```
I think is what you wanted to say?